deployments/docker-compose-definitions.yml

```diff
@@ -28,6 +28,8 @@ services:
     environment:
     - MEDCO_NODE_IDX=0
     - UNLYNX_DEBUG_LEVEL=1
+    volumes:
+      - ./dev-local-3nodes/configuration:/medco-configuration
```

Comment:
Is there a specific reason to have moved this config here from docker-compose.tools.yml? This file is shared by the different profiles.

Reply:
I agree with you it should not be there, it should be where it was before...but for some reason, the container was not able to access the files that way...I investigated a little bit, but with no success...I noticed that in the old medco-deployment the volume is declared this way in the definition.yml, so I re-put it here and everything worked again...weird, but maybe we can think about it when updating also the test profile?

Reply:
Strange... OK works for me.